storage/client/src/main/java/com/azure/storage/file/implementation/ServicesImpl.java

```diff
@@ -55,11 +58,12 @@ public ServicesImpl(AzureFileStorageImpl client) {
      * proxy service to perform REST calls.
      */
     @Host("{url}")
+    @Service("Storage File Services")
     private interface ServicesService {
         @PUT("")
         @ExpectedResponses({202})
         @UnexpectedResponseExceptionType(StorageErrorException.class)
-        Mono<ServicesSetPropertiesResponse> setProperties(@HostParam("url") String url, @BodyParam("application/xml; charset=utf-8") StorageServiceProperties storageServiceProperties, @QueryParam("timeout") Integer timeout, @HeaderParam("x-ms-version") String version, @QueryParam("restype") String restype, @QueryParam("comp") String comp, Context context);
+        Mono<ServicesSetPropertiesResponse> setProperties(@HostParam("url") String url, @BodyParam("application/xml; charset=utf-8") FileServiceProperties fileServiceProperties, @QueryParam("timeout") Integer timeout, @HeaderParam("x-ms-version") String version, @QueryParam("restype") String restype, @QueryParam("comp") String comp, Context context);
```

Comment:
What's the purpose of changing to fileServiceProperties?

Reply:
The spec calls for that to be the return type. StorageServiceProperties is a final class and I didn't think creating a duplicate class just for the return felt right so I renamed the class.

Reply:
Why does the storageServiceProperties cannot be used in this case? Does the property miss any fields?